docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FieldAwareFactorizationMachine.tt

```diff
@@ -0,0 +1,40 @@
+﻿<#@ include file="MultipleFeatureColumnsBinaryClassification.ttinclude"#>
+<#+
+string ClassName="FieldAwareFactorizationMachine";
+string Trainer = @"FieldAwareFactorizationMachine(
+                // Specify three feature columns!
```

Comment:
>// Specify three feature columns! [](start = 16, length = 33)

Majestic.
Maybe it's worth to mention what most of the trainers support one feature column, and why person should pass three separate feature columns instead of concat them into one.
Or maybe it's should be part of xml documentation and not related to you PR, what do I know about documentation anyway.

Reply:
This is explained in data generation code.

---
In reply to: [275450376](https://github.com/dotnet/machinelearning/pull/3312#discussion_r275450376) [](ancestors = 275450376)